Doc/library/sqlite3.rst

```diff
@@ -263,6 +263,7 @@ Module functions
                       autocommit=sqlite3.LEGACY_TRANSACTION_CONTROL)
 
    Open a connection to an SQLite database.
+   If *autocommit* is ``False``, a new transaction is implicitly opened.
```

Comment:
I think we should move this down to where the *autocommit* parameter is documented. @CAM-Gerlach, thoughts?

Reply:
I agree. Done.